frontend/src/ui/components/CollapseEditArea/CollapseEditArea.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Collapse, Nav } from 'react-bootstrap';
+import { WrappedMessage } from 'utils/intl';
+import messages from './displayMessages';
+import './styles.scss';
+
+interface CollapseEditAreaProps {
+  children: React.ReactNode;
+  initialExpanded?: boolean;
+}
+
+export const CollapseEditArea: React.FC<CollapseEditAreaProps> = (
+  props: CollapseEditAreaProps
+) => {
+  const [open, setOpen] = React.useState(props.initialExpanded || false);
+
+  return (
+    <div className="collapsible-edit-area">
+      <div className="edit-collapse">
+        <Nav.Link onClick={() => setOpen(!open)}>
+          <WrappedMessage messages={messages} id="edit" />
+          {!open && <i className="fas xs fa-chevron-up" />}
+          {open && <i className="fas xs fa-chevron-down" />}
```

Comment:
IMHO it'd be clearer to use an "if" statement here instead of the && logical operator.